src/cfd/constants/constants.tsx

```diff
@@ -250,3 +251,20 @@ export const AppToIconMapper: Record<string, ComponentType<SVGAttributes<SVGElem
     huawei: InstallationHuaweiIcon,
     ios: InstallationAppleIcon,
 };
+
+export const getWebtraderUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    return `${details?.white_label_links?.webtrader_url}?login=${details?.display_login}&server=${details?.server_info?.environment}`;
+};
+
+export const getDeeplinkUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    return `metatrader5://account?login=${details?.display_login}&server=${details?.server_info?.environment}`;
+};
+
+export const getMobileAppInstallerUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    if (mobileOsDetect() === 'iOS') {
+        return details?.white_label_links?.ios;
+    } else if (mobileOsDetect() === 'huawei') {
+        return 'https://appgallery.huawei.com/#/app/C102015329';
```

Comment:
Why this link is hard coded?

Reply:
unfortunately we are not getting all the required links needed from `white_label_links` 
will raise this